shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -2299,14 +2298,14 @@ public static WireCommand readFrom(EnhancedByteBufInputStream in, int length) th
             TableEntries entries = TableEntries.readFrom(in, in.available());
             boolean shouldClear = in.readBoolean();
             boolean reachedEnd = in.readBoolean();
-            long lastVersion = in.readLong();
+            long lastPosition = in.readLong();
 
-            return new TableEntriesDeltaRead(requestId, segment, entries, shouldClear, reachedEnd, lastVersion);
+            return new TableEntriesDeltaRead(requestId, segment, entries, shouldClear, reachedEnd, lastPosition);
```

Comment:
This is where you need to add it. See other cases in this class for examples how to do it.

If you want to know what, then follow through the `.release` call at line 2308 and see which `ByteBuf`s get released. Without invoking that (and requiring it), those buffers will stick around for a long time and lead to memory leaks).